### PR TITLE
attach points

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: content/creator/sdk7/3d-essentials/entity-positioning.md [hunk 1]
```diff
@@ -395,15 +395,43 @@ When creating an `AvatarAttach` component, pass an object with the following dat
...
+- `AAPT_RIGHT_HAND`: Fixed on the player's right hand
+- `AAPT_LEFT_HAND`: Fixed on the player's left hand
+- `.AAPT_NAME_TAG`: Floats right above the player's name tag, isn't affected by the player's animations.
 
   > Note: The name tag height is dynamically adjusted based on the height of the wearables a player has on. So a player wearing a tall hat will have their name tag a little bit higher than others.
 
-- `AvatarAnchorPointType.AAPT_POSITION`: The player's overall position.
-- `AvatarAnchorPointType.AAPT_RIGHT_HAND`: Is fixed on the player's right hand
-- `AvatarAnchorPointType.AAPT_LEFT_HAND`: Is fixed on the player's left hand
+- `AAPT_POSITION`: The player's overall position.
```
Comment (AlejandroAlvarezMelucciDCL) [inline content/creator/sdk7/3d-essentials/entity-positioning.md]: Maybe we can clarify that the position is in the ground level between the feet? And maybe also point out that currently there's a know issue where it's below the floor but will be addressed soon?

File: content/creator/sdk7/3d-essentials/entity-positioning.md [hunk 2]
```diff
@@ -395,15 +395,43 @@ When creating an `AvatarAttach` component, pass an object with the following dat
...
+- `AAPT_HEAD`: Fixed to center of the player's head.
+- `AAPT_NECK`: Fixed to the player's base of the neck.
+- `AAPT_SPINE`: Fixed to the top section of the backbone.
```
Comment (AlejandroAlvarezMelucciDCL) [inline content/creator/sdk7/3d-essentials/entity-positioning.md:404]: Maybe we can add that creators might want to add an offset when using this bone for positioning objects on the back or on the chest? And later on we could add a "Suggested offsets" table with some values for these and others?

**Review by AlejandroAlvarezMelucciDCL — APPROVED** [review]
Approved but left 2 small suggestions :)

File: content/creator/sdk7/3d-essentials/entity-positioning.md (hunk 2 above)
Comment (nearnshaw) [reply content/creator/sdk7/3d-essentials/entity-positioning.md:404]: this is clarified and there's a snippet showing how to do this just a few lines below

A list of suggested offsets would be great, we need to come up with these values later

File: content/creator/sdk7/3d-essentials/entity-positioning.md (hunk 1 above)
Comment (nearnshaw) [reply content/creator/sdk7/3d-essentials/entity-positioning.md]: added more detail now

**Pull request merged**